R/survival.R

```diff
@@ -624,9 +637,9 @@ get_migratory_survival_rates <- function(year, month,
                                          ..surv_juv_outmigration_sac_prop_diversions = ..surv_juv_outmigration_sac_prop_diversions,
```

Comment:
why `..surv_juv_outmigration_sac_prop_diversions = ..surv_juv_outmigration_sac_prop_diversions,` instead of just `..surv_juv_outmigration_sac_prop_diversions, `? for all the arguments here?